rust/ql/test/utils-tests/modelgenerator/CaptureSummaryModels.expected

```diff
@@ -1,2 +1,4 @@
 unexpectedModel
 expectedModel
+| Expected summary missing: repo::test;<crate::option::MyOption as crate::clone::Clone>::clone;Argument[self].Field[test::option::MyOption::MySome(0)].Reference;ReturnValue.Field[test::option::MyOption::MySome(0)];value;dfc-generated |
+| Expected summary missing: repo::test;<crate::option::MyOption>::cloned;Argument[self].Field[test::option::MyOption::MySome(0)].Reference;ReturnValue.Field[test::option::MyOption::MySome(0)];value;dfc-generated |
```

Comment:
These models look correct to me.  If you agree, I think you're supposed to add them as comments in `option.rs` so they become expected results.